modules/votes/ui/VoteDetails/VoteDetails.tsx

```diff
@@ -1,37 +1,40 @@
 import { Text } from '@lidofinance/lido-ui'
-import { FormattedDate } from 'modules/shared/ui/Utils/FormattedDate'
+import type {
+  AttemptCastVoteAsDelegateEventObject,
+  CastVoteEventObject,
+} from 'generated/AragonVotingAbi'
 import { VoteScript } from '../VoteScript'
-import { VoteDetailsCountdown } from '../VoteDetailsCountdown'
-import { VoteStatusBanner } from '../VoteStatusBanner'
-import { VotePhasesTooltip } from '../VotePhasesTooltip'
+import { VoteStatusBanner as VoteStatusBannerNew } from 'modules/votes/ui/VoteStatusBannerNew'
```

Comment:
`VoteStatusBannerNew` seems like it was temporary naming, reminding to update it back with `VoteStatusBanner`